gateway-ha/src/main/java/io/trino/gateway/ha/config/HaGatewayConfiguration.java

```diff
@@ -44,6 +44,18 @@ public class HaGatewayConfiguration
 
     private RequestAnalyzerConfig requestAnalyzerConfig = new RequestAnalyzerConfig();
 
+    private UIConfiguration uiConfiguration = new UIConfiguration();
+
+    public UIConfiguration getUiConfiguration()
+    {
+        return uiConfiguration;
+    }
+
+    public void setUiConfiguration(UIConfiguration uiConfiguration)
+    {
+        this.uiConfiguration = uiConfiguration;
+    }
```

Comment:
Move these getter and setter under `setRequestAnalyzerConfig` method.

Reply:
Sorry I don't think I understand why should we move this into setRequestAnalyzerConfig? I believe that serves a different purpose and this class I added so that we can control things on the UI like enabling/disabling pages.

Reply:
I mean moving this method in this class.